osc-server/src/test/java/org/osc/core/broker/service/servicefunctionchain/UpdateServiceFunctionChainServiceTest.java

```diff
@@ -60,91 +60,77 @@ public void testDispatch_WithNullRequest_ThrowsNullPointerException() throws Exc
 	@Test
 	public void testDispatch_WithNonEmptyListOfVsIdsUpdateValidationSuccessful_ValidationSucceeds() throws Exception {
 		
+		//Arrange
 		BaseDto dto = new BaseDto();
 		dto.setId(this.sfc.getId());
 		dto.setParentId(this.vc.getId());
 		request.setDto(dto);
 		request.setName(this.sfc.getName());
 		List<Long> vsIds = new ArrayList<Long>();
+		
 		vsIds.add(this.vs.getId());
 		vsIds.add(this.vs1.getId());
 		request.setVirtualSystemIds(vsIds);
 		Mockito.when(this.service.validator.validateAndLoad(request)).thenReturn(this.sfc);
+		
 		// Act.
 		BaseJobResponse response = this.service.dispatch(request);
+		
+		//Assert
 		Assert.assertNotNull("The returned response should not be null.", response);
```

Comment:
Also please validate what should be in the response.

Reply:
added response id check